docs/class-definitions.md

```diff
@@ -0,0 +1,73 @@
+## Class Definitions
+
+**Contents**
+
+- [Spaces](#spaces)
+- [SpaceOptions](#spaceoptions)
+- [Space](#space)
+- [SpaceMember](#spacemember)
+
+### Spaces
+
+#### constructor(ably)
+
+Create a new instance of the Spaces library.
+This requires a promise instance of the Ably library to be passed in.
+Refer to the [Ably-JS Documentation](https://github.com/ably/ably-js#introduction) for information on setting up a realtime promise client.
+
+| Property | Type                 |
+|----------|----------------------|
+| ably     | Ably.RealtimePromise |
+
+
+#### get(name, options?)
+Get or create a space. Options may only be provided if a space does not yet exist.
+
+| Property | Type                                           |
+|----------|------------------------------------------------|
+| name     | string                                         |
+| options  | [SpaceOptions](#spaceoptions) &#124; undefined |
+
+### SpaceOptions
+
+Used to configure a Space object on creation.
+
+#### offlineTimeout
+Number of milliseconds after a user loses connection or closes their browser window to wait before their [SpaceMember](#spacemember) object is removed from the members list.
+
+Default is 5000ms.
```

Comment:
The default is `120_000` https://github.com/ably-labs/spaces/blob/main/src/Space.ts#L26